test/generators.jl

```diff
@@ -34,6 +34,9 @@ g = Grid([3,3,4])
 @test maximum(degree(g)) == 6
 @test minimum(degree(g)) == 3
 
+g = CliqueGraph(3,5)
+@test nv(g) == 15 && ne(g) == 20
```

Comment:
This may be an unfair request, but can you test a specific property of the cliquegraph besides just `nv` and `ne`? That is, can you extract a clique (using induced subgraphs) and compare it against a CompleteGraph of the same size?